examples/incubating.md

```diff
@@ -1323,21 +1323,21 @@ displayed on the light grey of tables.*
     </tr>
     <tr>
         <th>black</th>
```

Comment:
Apart from black and white, we need to update the "color" version as well. We can use the black version to replace the "color".